pkg/cli/cmd/radinit/azure_test.go

```diff
@@ -82,6 +86,58 @@ func Test_enterAzureCloudProvider(t *testing.T) {
 	require.Equal(t, expectedOutput, outputSink.Writes)
 }
 
+func Test_enterAzureCloudProvider_WorkloadIdentity(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	prompter := prompt.NewMockInterface(ctrl)
+	client := azure.NewMockClient(ctrl)
+	outputSink := output.MockOutput{}
+	runner := Runner{Prompter: prompter, azureClient: client, Output: &outputSink}
+
+	subscription := azure.Subscription{
+		Name: "test-subscription",
+		ID:   "test-subscription-id",
+	}
+
+	resourceGroup := armresources.ResourceGroup{
+		Name: to.Ptr("test-resource-group"),
+	}
+
+	// selectAzureSubscription
```

Comment:
Is there a reason why the comments are camel case?

Reply:
looks like they aren't very useful - removed